cmd/kubeadm/app/master/manifests.go

```diff
@@ -367,6 +362,27 @@ func getAPIServerCommand(cfg *kubeadmapi.MasterConfiguration, selfHosted bool) [
 	return command
 }
 
+func getEtcdCommand(cfg *kubeadmapi.MasterConfiguration) []string {
+	defaultURL := "http://127.0.0.1:2379"
+
+	listenURL := defaultURL
```

Comment:
Please use the same logic as the other control plane components here with default and overridable arguments.